### PR TITLE
api: remove Chromium* classes

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/client/browserContext.ts [hunk 1]
```diff
@@ -59,11 +65,23 @@ export class BrowserContext extends ChannelOwner<channels.BrowserContextChannel,
     super(parent, type, guid, initializer);
     if (parent instanceof Browser)
       this._browser = parent;
+    this._isChromium = this._browser?._name === 'chromium';
```
Comment (dgozman) [inline src/client/browserContext.ts:68]: I think we should have `initializer.isChromium` instead, which is also set for Electron and Android.

File: types/types.d.ts [hunk 2]
```diff
@@ -1484,10 +1484,11 @@ export interface Page {
   context(): BrowserContext;
 
   /**
-   * Browser-specific Coverage implementation, only available for Chromium atm. See
-   * [ChromiumCoverage](#class-chromiumcoverage) for more details.
+   * > NOTE: Only available for Chromium atm.
```
Comment (dgozman) [inline types/types.d.ts:1487]: Can we rephrase to not mention "Automated Teller Machine"? 😄

**Review by dgozman — APPROVED** [review]

File: src/client/browserContext.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/client/browserContext.ts:68]: i'll follow up

File: types/types.d.ts (hunk 2 above)
Comment (pavelfeldman) [reply types/types.d.ts:1487]: I'd rather not focus on it atm :P

**Pull request merged**